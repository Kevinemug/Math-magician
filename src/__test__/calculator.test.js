import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../components/calculator';

it('renders Calculator correctly', () => {
  const tree = renderer.create(<Calculator />);
  expect(tree).toMatchSnapshot();
});
