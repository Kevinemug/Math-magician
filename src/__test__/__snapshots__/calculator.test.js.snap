// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Calculator correctly 1`] = `
<div
  className="calc-container"
>
  <div
    className="displayscreen-cont"
  >
    <span
      className="display-screen"
    />
  </div>
  <button
    className="btn btn-top"
    onClick={[Function]}
    type="button"
  >
    AC
  </button>
  <button
    className="btn btn-top"
    onClick={[Function]}
    type="button"
  >
    +/-
  </button>
  <button
    className="btn btn-top"
    onClick={[Function]}
    type="button"
  >
    %
  </button>
  <button
    className="btn btn-side"
    onClick={[Function]}
    type="button"
  >
    รท
  </button>
  <button
    className="btn btn-side"
    onClick={[Function]}
    type="button"
  >
    x
  </button>
  <button
    className="btn btn-side"
    onClick={[Function]}
    type="button"
  >
    -
  </button>
  <button
    className="btn btn-side"
    onClick={[Function]}
    type="button"
  >
    +
  </button>
  <button
    className="btn btn-side"
    onClick={[Function]}
    type="button"
  >
    =
  </button>
  <div
    className="numbers"
  >
    <button
      className="btn btn-primary"
      data-testid="seven"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="btn btn-primary"
      data-testid="eight"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="btn btn-primary"
      data-testid="nine"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="btn btn-primary"
      data-testid="four"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="btn btn-primary"
      data-testid="five"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="btn btn-primary"
      data-testid="six"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="btn btn-primary"
      data-testid="one"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="btn btn-primary"
      data-testid="two"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="btn btn-primary"
      data-testid="three"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="btn btn-primary col-span-2"
      data-testid="zero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="btn btn-primary"
      data-testid="full-stop"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
  </div>
</div>
`;
